readme.md

```diff
@@ -0,0 +1,40 @@
+﻿# SC Duty List Web Page
+
+The SC (Student Councillor for short) Duty List Web Page is for Student Councillors to use to easily check for their duty spots. 
+
+The website is meant for private use but I am making it public in good faith that it will help others who are looking for examples for these kinds of projects.
+
+If you need anything, I am contactable through the following email davian@davianeng.com.
+
+# The Design Process
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8390593)
```suggestion
# The Design Process

```